install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -130,7 +130,7 @@ data:
     outboundTrafficPolicy:
       mode: {{ .Values.global.outboundTrafficPolicy.mode }}
 
-    {{- if  .Values.global.localityLbSetting }}
+    {{- if  .Values.global.localityLbSetting.enabled }}
     localityLbSetting:
 {{ toYaml .Values.global.localityLbSetting | indent 6 }}
```

Comment:
nit: should we separate the localityLbSetting content and enablement flag? do we really want the enablement flag(`enabled: true`) in the final `localityLbSetting` yaml?